pages/clustering/replication.mdx

```diff
@@ -126,6 +126,22 @@ The MAIN's UUID is also stored on a disk, in case of restart of an instance to c
 MAIN instance. When REPLICA restarts, `--replication-restore-state-on-startup` must be set to 
 `true` to continue getting updates from the MAIN.
 
+## Auth data replication (Enterprise)
+
+If you are using a Memgraph Enterprise license, all authentication/authorization data, including users, roles, and associated permissions, will be replicated. 
+However, authentication modules are not replicated and must be configured manually by the administrator.
+
+## Multi-tenant data replication (Enterprise)
+
+If you are using a Memgraph Enterprise license multi-tenant commands are replicated as any other data command.
+Database manipulation is allowed only on MAIN. However, REPLICAs have the
+ability to use databases and read data contained in them.
+
+When dropping a database used on a REPLICA, the REPLICA will receive the command
+and will partially drop the database. It will hide it and not allow any new
+usage of the database but will wait for every client to release the database
```

Comment:
Remove "but" and start a new sentence. Something like "Once all clients have released the database, then it will be deleted"